src/api/treeTrackerApi.js

```diff
@@ -120,6 +121,7 @@ export default {
   async rejectCaptureImage(capture, rejectionReason) {
     try {
       log.debug('reject capture', capture.id, rejectionReason);
+      console.log('env url', FIELD_DATA_API);
```

Comment:
Please don't leave `console.log()` statements in the code. If you believe they are useful for debug, use `log.debug()` instead, as it is switched off by default in production but can be enabled if need be.